cli/template/extras/src/server/db/drizzle-schema-base.ts

```diff
@@ -0,0 +1,34 @@
+// Example model schema from the Drizzle docs
+// https://orm.drizzle.team/docs/sql-schema-declaration
+
+import { sql } from "drizzle-orm";
+import {
+  mysqlTableCreator,
+  serial,
+  timestamp,
+  uniqueIndex,
+  varchar,
+} from "drizzle-orm/mysql-core";
+
+/**
+ * This is an example of how to use the multi-project schema feature of Drizzle ORM. Use the same
+ * database instance for multiple projects.
+ *
+ * @see https://orm.drizzle.team/docs/goodies#multi-project-schema
+ */
+export const mysqlTable = mysqlTableCreator((name) => `project1_${name}`);
+
+export const example = mysqlTable(
+  "example",
+  {
+    id: serial("id").primaryKey(),
```

Comment:
We will change the documentation examples for MySQL to use `bigint().primaryKey().autoincrement()` instead of serial. Using SERIAL may not explicitly convey the use of a bigint autoincrement field and could lead to issues. We will also make the same adjustment for PostgreSQL, especially since PostgreSQL is beginning to recommend IDENTITY columns over SERIAL column types. While we won't deprecate these options, we don't want to feature them in our examples or promote them as the recommended way to set up a schema

So it would be great, if create-t3-app will also recommend this id columns:
```ts
id: bigint("id", { mode: "number" }).primaryKey().autoincrement()
```
We will update drizzle orm docs with this recommendation as well

Reply:
I changed it to use bigint, would love for unsigned support in the near future though 🙏🏼

Reply:
These recommendations are intended for new users, which is why we will update our documentation for now (while there is no unsigned option). Furthermore, the unsigned option should be included in the next release, so in any case, it will be available soon.

I'm curious about why you want to have the unsigned option and why bigint alone is not sufficient. BIGINT has a range of up to 9,223,372,036,854,775,807. I understand that with unsigned, you can extend this range by a factor of two, but even with a signed option, I can imagine that there may be so many rows in a table that reaching 9,223,372,036,854,775,807 would be challenging.

Consider, for example, a messages table receiving 10 billion messages per day. In this scenario, it would take approximately 2,479,401 years to reach the limits. This calculation assumes we are discussing changing from serial to bigint

Reply:
Yea obviously a BIGINT will be enough for most apps, but I just like being explicit what data will be in the column, and since we won't have any negatives it just makes sense that it's unsigned...

Reply:
Agree! 
Will add support for it soon!

Reply:
I think we're good to merge this then!

Exciting, app dir next :)